app/src/main/java/com/hover/stax/channels/ChannelDropdown.java

```diff
@@ -94,6 +95,11 @@ private void onSelect(Channel c) {
 		setDropdownValue(c);
 		if (highlightListener != null) { highlightListener.highlightChannel(c); }
 		highlightedChannel = c;
+		if(c!=null) {
+			FirebaseMessaging.getInstance().subscribeToTopic(
```

Comment:
This should be in the logChoice function in the ChannelDropdownViewModel

Reply:
Yup, will update, thanks.